src/main/java/frc/robot/utils/motors/FalconConstants.java

```diff
@@ -0,0 +1,11 @@
+package frc.robot.utils.motors;
+
+public class FalconConstants {
```

Comment:
1. Isn't a MiniCIM used in the prototype?
2. You can simply use [`DCMotor.getMiniCIM()`](https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/math/system/plant/DCMotor.html#getMiniCIM(int)) for all of these data.